data_steward/cdr_cleaner/manual_cleaning_rules/remediate_basics.py

```diff
@@ -343,14 +344,16 @@ def setup_rule(self, client):
         will have NO duplicate. Each destination dataset can have different new
         observation_ids so this lookup table has `dataset_id` column.
         """
-        create_lookup = self._get_query(CREATE_NEW_OBS_ID_LOOKUP, '', '')[QUERY]
-        job = client.query(create_lookup)
-        job.result()
+        if not client.table_exists(NEW_OBS_ID_LOOKUP, self.sandbox_dataset_id):
```

Comment:
@hiro-mishima This block of code says it creates the lookup table if it does not exist, this should be modified to creating lookup in case of combined_release and in other cases copyin the created lookup in combined_release.

Reply:
@ksdkalluri Based on this comment, I am making only one lookup table and that will be referenced by all the datasets for r9 and r10. https://github.com/all-of-us/curation/pull/1485#discussion_r1106036945

Reply:
Thanks for pointing it out and for the quick huddle, @ksdkalluri!
I updated dataset references to the correct ones: https://github.com/all-of-us/curation/pull/1489/commits/92f4c7bd84d03faa5d0e273723076ba51025c891